.gitignore

```diff
@@ -23,4 +23,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-
+# build products
+src/**/*.css
```

Comment:
@BoltC0rt3z why add this to `.gitignore`? I don't think we will be having any `.css` file since we are using `sass`

Reply:
From the implementation of sass, It creates .css file when you run the app, so that's the reason.
I referenced it from [here](https://medium.com/stephenkoo/how-to-set-up-create-react-app-redux-react-router-redux-thunk-prettier-scss-airbnb-eslint-dda0bba5616a)